RevenueCatUI/CustomerCenter/Data/CustomerCenterActionHandler.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import RevenueCat
+
+@objc
+public protocol CustomerCenterActionHandler {
+    @objc optional func purchaseCompleted(_ customerInfo: CustomerInfo)
+    @objc optional func restoreStarted()
+    @objc optional func restoreCompleted(_ customerInfo: CustomerInfo)
```

Comment:
I also thought about having a single method here and a enum for the different events, but this seems simpler and less prone to breaking changes IMO. Lmk what you think!